platformio.ini

```diff
@@ -37,6 +37,8 @@ lib_deps =
 	bodmer/TJpg_Decoder@^1.1.0
 	paulstoffregen/Time@^1.6.1
 	arduino-libraries/NTPClient @ ^3.2.1
+	bblanchon/StreamUtils@^1.9.0
```

Comment:
Not the biggest fan of more libraries since we are running tight on space but i guess it's fine

Reply:
Me neither, but Parqet uses chunked responses and as far as I know, HTTPClient does not support this natively. 

Chunked responses will probably also be needed for other webservices in the future, therefore I think it's not too bad to include the library.

Reply:
Fair enough